src/auth-service/utils/email.msgs.js

```diff
@@ -40,6 +40,56 @@ module.exports = {
     );
     // return `${message} \n\n` + `https://airqo.net/`;
   },
+
```

Comment:
First version of email templates, same content as the one below....just that this one does not carrying any "styling"......but all the same thing.